src/components/DataBrowser.vue

```diff
@@ -19,8 +20,10 @@ export default {
     },
     location: {
       type: Object,
-      required: true,
-      validator: val => val._modelType && val._id,
+      default: () => ({
+        type: 'root',
+      }),
+      validator: getLocationValidator(true),
```

Comment:
I kind of liked the symmetry of inputting a model object (with `_modelType` and `_id`) as a prop and receiving similar model objects on event callbacks. Could we keep the same API for the prop as before and just have special values of "root", "users", and "collections" for the `_modelType`, where the `_id` would be ignored in those cases?

Reply:
Hm, I see the convenience of just passing a girder model to location and the DataBrowser will navigate to it, but I think putting _modelType and these virtual things "root", "users", and "collections" is not a good thing to do. How about the location prop could take either a location object `{location: 'root'}`, or a 'user', 'folder', 'collection' model, but for the sake of consistency and to avoid confusion, I think the 'update: location' should always report `location` object. (we can now not listen to it if we don't need to know the location.) Since the 'itemclick' and 'selection-changed' is the event for objects?

Reply:
I agree with Jeff, and would like to keep this interface the same.  It's quite nice for upstream to be able to pass responses directly from girder API requests into GWC.  I don't think consumers should have to do mapping to make these components happy.

Reply:
@subdavis I've talked to @matthewma7 about this just now, and I'm ok either way and retract my opinion. Basically, @matthewma7's implementation is inconsistent with prior behavior and other model objects floating around in a Girder client app, but your suggestion is inconsistent with what "_modelType" means, in that there is no "root", etc. model in Girder, which may be confusing to people looking at REST endpoints and server code. Since I don't think we can win here, I'm ok with either approach.

Reply:
Hi @subdavis, I just had an offline discussion with @jeffbaumes. He agreed that putting _modelType: 'root' is confusing for people who are not as familiar with girder. This location is purely a GWC concept and keeping it explicit make it easier to understand. But to make it more convenient, I would update to make the location prop be able to take a user, collection, folder model as input, so the user of the component doesn't have to do the mapping.

Reply:
@matthewma7 sure, if we can hide the mapping such that this isn't a breaking change, I'm happy with that.  Could you make that change in both DataBrowser and Breadcrumb? ty.

Reply:
Yes. I will make both of them can take girder model as the initial value.